vertx/pom.xml

```diff
@@ -24,16 +24,22 @@
     <version>13.6-SNAPSHOT</version>
   </parent>
 
-  <artifactId>feign-vertx</artifactId>
+  <artifactId>feign-vertx-parent</artifactId>
+  <packaging>pom</packaging>
 
-  <name>Feign Vertx</name>
-  <description>Implementation of Feign on Vertx web client.</description>
+  <modules>
+    <module>feign-vertx</module>
+    <module>tests-vertx4</module>
+    <module>tests-vertx-common</module>
+    <module>tests-vertx5</module>
+  </modules>
 
   <properties>
-    <vertx.version>4.5.12</vertx.version>
-    <jackson.version>2.18.2</jackson.version>
-    <slf4j-log4j12.version>2.0.0-alpha6</slf4j-log4j12.version>
-    <wiremock.version>2.35.2</wiremock.version>
+    <main.java.version>11</main.java.version>
```

Comment:
is this needed?!

11 is the min version already

Reply:
Will check for it. Thanks